awscli/examples/mediaconvert/describe-endpoints.rst

```diff
@@ -0,0 +1,15 @@
+**To get your account-specific endpoint**
+
+The following example gets the endpoint that you need to send any other request to the service:
+
+Command::
+     aws mediaconvert describe-endpoints
```

Comment:
Ah you do have an example of this. You could link to it from the other ones to make it easier to find.

Reply:
Sounds like a great idea! Could you tell me how?

Reply:
@EFolsom , we have instructions on how to create RST links in the wiki - see the bit right before the header "Create the Examples" on this page: https://w.amazon.com/bin/view/AWSDocs/common-tasks/create-examples/create-aws-cli-examples/#HCreatetheexamples

Reply:
Sure, I get how to put a link into rst. My questions are:
* how do I know what the url will be once it's merged in and published?
* how is it going to look when someone accesses the help at the command line?